Makefile

```diff
@@ -7,7 +7,7 @@ GOBIN?=${GOPATH}/bin
 PACKAGES=$(shell go list ./...)
 OUTPUT?=build/tendermint
 
-INCLUDE = -I=. -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
+INCLUDE = -I=${GOPATH}/src/github.com/tendermint/tendermint -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
```

Comment:
`.` does not work?

Reply:
It doesn't work because the compilation is now moved at the `GOPATH/src` level to keep the naming from the `go_package` option.

Generated files are generated using the full path of the go_package and will then generate a new `github.com/tendermint/tendermint/packageX` within the sources. 

That's why the compilation is not on the root of the project but the root of the `GOPATH/src`.

This can be changed by changing the `go_package` option from `github.com/tendermint/tendermint/packageX` to `packageX`. It doesn't follow go conventions but it's totally acceptable.

If it's done this way, the makefile will not change and the compilation will still be at the project level.

Let me know if you prefer this solution.

Reply:
I see. Thanks for explaining it to me! We should probably play nice and respect the naming.

Reply:
Do you want me to revert the last commit?

Reply:
Please do

Reply:
done :)